src/Gnugat/Medio/Model/Method.php

```diff
@@ -200,4 +215,39 @@ public function getBody()
     {
         return $this->body;
     }
+
+    /**
+     * @return $this
+     */
+    public function makeAbstract()
+    {
+  
+        if ($this->isStatic()) {
+            throw new \Gnugat\Medio\Exception\DomainException("You can`t declare static method as abstract: " . $this->getName());
+        }
+
+        if ($this->getBody()) {
+            throw new \Gnugat\Medio\Exception\DomainException("Method contain body. You can`t declare method as abstract: " . $this->getName());
```

Comment:
You could add a use statement to avoid having to write the exception's fully qualified classname (we'd be able to see the whole message without scrolling, and by looking at the top of the file we'd be able to tell the dependencies).

Coding standard: no spaces around the concatenation operator